lib/preface_make/meet_semilattice.ml

```diff
@@ -0,0 +1,23 @@
+module Core (Req : Preface_specs.Meet_semilattice.WITH_MEET) = Req
+
+module Infix (Core : Preface_specs.Meet_semilattice.CORE) = struct
+  type t = Core.t
+
+  let ( ^ ) = Core.meet
```

Comment:
While I understand the desire to use `^`, it might cause a symmetry problem with `Join_semilattice`. So I suggest using `&` (or `&&`). What do you think @hakimba?

Reply:
I understand the symmetry issue so I have no problem with your solution !

Reply:
@gr-im, @hakimba what do you prefer, `&` or `&&` ?

Reply:
Fixed on https://github.com/xvw/preface/pull/178/commits/4424d4131ca33c4842dfa89b6ad772f04f6b0dea